### PR TITLE
added new JS question Q5

### PR DESCRIPTION
**Pull request by Dhanu084**

### CODE REVIEW

File: javascript/answers/q5.html [hunk 1]
```diff
@@ -0,0 +1,76 @@
...
+        gtag('config', 'UA-157837769-1');
+        </script>
+        
+        <title>
+            JavaScript | InterQues | Coding Ninjas
+        </title>
+
+        <link rel="shortcut icon" type="image/png" href="../../assets/favicon.png"/>
+
+        <meta name="description" content="What will be the output of the code below?">
```
Comment (arpansac) [inline javascript/answers/q5.html]: Please change this description to the question text!

**Review by arpansac — CHANGES_REQUESTED** [review]
Added a comment!

File: javascript/answers/q5.html (hunk 1 above)
Comment (Dhanu084) [reply javascript/answers/q5.html]: Done sir

**Review by arpansac — APPROVED** [review]
looks cool

**Pull request merged**